portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,14 +26,26 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles fetching and posting of comments */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<Comment> comments = new ArrayList<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", Query.SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
How about putting this as an instance field?

Reply:
ah yes that would also reduce code repetition.